app/javascript/components/concept-map/Concept.tsx

```diff
@@ -80,10 +81,12 @@ export const Concept = ({
         onMouseLeave={wrapAnimationFrame(handleLeave)}
       >
         <div className="display">
-          <ConceptIcon name={name} size="small" />
+          <ConceptIcon name={name} size="medium" />
           <div className="name">{name}</div>
         </div>
-        {!isLocked && <PureExerciseStatusBar {...exercises} />}
+        {!isLocked && (
+          <PureExerciseStatusBar exerciseStatuses={exerciseStatuses} />
```

Comment:
This is one of the places where I'm not sure whether I did the right or wrong thing. But I couldn't work out how to get it to work the way you had it so I changed it to what I know.